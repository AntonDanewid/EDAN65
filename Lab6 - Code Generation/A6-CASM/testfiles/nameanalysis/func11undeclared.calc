int a() {
	return b(); /* error: b is not yet declared !!DECLARATION ORDER!! */
}
int b() {
	return 1;
}

/* FEL ORDNING PÃ… METODANROP */