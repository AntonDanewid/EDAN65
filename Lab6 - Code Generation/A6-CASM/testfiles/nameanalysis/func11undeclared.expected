Error at line 2: function 'b' is not declared
